source/clear-linux/guides/deploy-at-scale.rst

```diff
@@ -0,0 +1,287 @@
+.. _deploy-at-scale.rst:
+
+Deploying |CL| at Scale
+#######################
+
+Once you are comfortable with `Clear Linux concepts`_, your next step
+as a system administrator is to understand how to deploy Clear Linux*
+at scale in your environment.  
+
+In this document the term *endpoint* refers to a system targeted for
+|CL| installation, whether that is a datacenter system or unit deployed in
+field.  
+
+.. note::
+
+    This is not a replacement or blueprint for designing your own IT
+    operating environment. 
+    
+    Your |CL| deployment should complement the existing environment and
+    available tools. It is assumed foundational core IT dependencies of your
+    environment, such as your network, are healthy and scaled to suit the
+    deployment.
+
+.. contents:: :local:
+    :depth: 2
+
+Pick a |CL| usage and update strategy
+=====================================
+
+Different business scenarios call for different deployment methodologies. 
+|CLOSIA| offers the flexibility to continue consuming the upstream |CL|
+distribution or the option to fork away from the |CL| distribution and
+act as your own :abbr:`OSV (Operating System Vendor)`. 
+
+Below are overviews of both approaches and some considerations.
+
+Option #1: Use the |CL| as the upstream origin (mixin)
+------------------------------------------------------
+
+This approach is *easier to adopt* by relying on the |CL| upstream for 
+packaging updates for you to deploy. 
+
+Custom software or packages that are not available in a preformed bundle
+can be added using the `mixin process`_ to form a custom bundle. 
+If custom bundles are needed, you will be responsible for maintaining 
+the custom bundle(s) and testing between |CL| releases in your environment, 
+while the rest of the operating system and preformed bundles come from the 
+|CL| upstream.
+    
+#. Ensure |CL| systems are able to be inventoried, managed, and orchestrated 
+   to coordinate software updates.
+
+#. With autoupdate enabled, |CL| is updated daily, however you may wish to
+   act as an intermediary buffer between the OS releases. If you do decide
+   to act as a gate to |CL| versions, define a desired release cadence for
+   yourself which is realistic with the operational expectations of your
+   environment.
+
+#. Use a web caching proxy for |CL| updates for devices connected to 
+   a local area network (LAN), like a datacenter, to increase the speed 
+   and resiliency of updates from the |CL| update servers. 
+   
+   Your caching proxy server is just like any other web application. 
+   |WEB-SERVER-SCALE|
```

Comment:
This is referenced 3 times.  The first two references use a period for the previous sentence.  the third uses a ;  Should these be consistent with each other?

Reply:
@puneetse can you take a look?